NEWS

```diff
@@ -12,6 +12,10 @@
   which have empty repository scopes (gh#SUSE/machinery#1268)
 * Inspection of unmanaged-files is now also using the faster machinery-helper
   when the files are extracted
+* Introduce format version 5 which adds the "environment" scope
```

Comment:
Maybe better 'internal "environment" scope' to make it clear that it's not supposed to be visible to users.